src/mongo/MongoActionReader.test.ts

```diff
@@ -41,6 +41,6 @@ describe('MongoActionReader', () => {
       dbName: 'failed',
     })
     const result = failedReader.getNextBlock()
-    expect(result).rejects.toThrow(NotInitializedError)
+    await expect(result).rejects.toThrow(NotInitializedError)
```

Comment:
I think you don't need an await here, though it may not hurt either.

Reply:
@jeffreyssmith2nd I was surprised too https://jestjs.io/docs/en/tutorial-async/ I'm thinking they finally fixed this with rejections and jet?

Reply:
The documentation you linked definitely seems like it should be fine this way. Unexpected, but works for me 👍

Reply:
I needed to lock versions in this cleanup, as there was a breaking bugfix release in @types/jest. In the process, I upgraded jest from 22 to 23, and I think in that version `toThrow` returns a promise. I needed to add the await or the build would fail.